### ISSUE TITLE
rewardHeight deployer option error

### ISSUE THREAD

**Issue by Nigui** [ArkEcosystem/deployer#37]
I'm using `deployer-v2` branch to deploy a bridgechain and I've got the following error message:

```
/home/ark/ark-deployer/datadir/packages/core-deployer/bin/deployer:213
network.constants[1].height = options.rewardHeight
                            ^

TypeError: Cannot set property 'height' of undefined
    at Object.<anonymous> (/home/ark/ark-deployer/datadir/packages/core-deployer/bin/deployer:213:31)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
    at startup (internal/bootstrap/node.js:285:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3) 
```
Config: 
- ubuntu 18.10 
- default config file (`startRewardHeight` at 75600)

My workaround was to remove the `--rewardHeight` option into [`deployer/app-node.sh`](https://github.com/ArkEcosystem/deployer/blob/deployer-v2/app/app-node.sh#L88)

**Comment by alexbarnsley** [comment]
Thank you for opening an issue. The deployer-v2 branch is still unfinished so there are likely errors. I will close this for now because it will likely get fixed once it's revisited and finished. It will be merged into the master branch once it is ready.
